### PR TITLE
Tko november 2021

### PR DESCRIPTION
**Pull request by tkoukkari**
* fix for ismodal:  considers timeouts
* updated getpdftext and getfiletext docs
* fix for SetConfig   CaseInsensitive

### CODE REVIEW

File: QWeb/keywords/file.py [hunk 1]
```diff
@@ -124,8 +127,8 @@ def get_file_text(**kwargs):
     --------
     .. code-block:: robotframework
 
-        ${text}      GetFileText     #returns whole content
-        ${text}      GetFileText     xyz   10  #returns 10 chars. Starting from text xyz.
+        ${text}    GetFileText    #returns whole content
+        ${text}    GetFileText    xyz   10  #returns 10 chars, tarting from text xyz.
```
Comment (Ekexyz) [inline QWeb/keywords/file.py]: s*

Comment (tkoukkari) [reply QWeb/keywords/file.py]: > 

oops

Comment (tkoukkari) [reply QWeb/keywords/file.py]: thanks, new commit fixes this

**Review by Ekexyz — APPROVED** [review]
LGTM

**Review by irosengr — APPROVED** [review]

**Pull request merged**